### ISSUE TITLE
Post Request Parameters issue

### ISSUE THREAD

**Issue by wanyakun** [loopj/android-async-http#770]
I write a method to call by other class as follow:
```java
    /**
     * POST请求
     * @param context
     * @param url
     * @param params
     * @param responseHandler
     */
    public static void post(Context context, String url, RequestParams params, BaseJsonHttpResponseHandler responseHandler) {
        client.setProxy("192.168.2.25",8888);
        params.setUseJsonStreamer(true);
        params.setAutoCloseInputStreams(true);
        client.addHeader("Content-Type", RequestParams.APPLICATION_JSON);
        client.post(context, url, params, responseHandler);

    }
```
The value of params as follow:
```
"queryType": "DM",
"entityCode": "2",
"channelType": "1"
```
When I use Charles to monitor the post request, i found all the requests were add a parameter as follow:
```
 "_elapsed": 0
```
This case all my request fail. I don't know is this a issue or I use this lib in a wrong way.  
Look forward to your reply. Thank a lot.

**Comment by fineswap** [comment]
@wanyakun 
It'll be added as a feature request, thanks for the heads up.

**Comment by fineswap** [comment]
@wanyakun 
This is now addressed in pull #780, which @smarek will decide whether it's fit for merging or not. Please keep an eye on that pull request.

**Comment by smarek** [comment]
@wanyakun @fineswap merged, consider this closed in latest snapshot
